src/index.html

```diff
@@ -6,9 +6,59 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#section-1" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        class="logo__img"
+        >
+      </a>
```

Comment:
apply code styling recommendations from a checklist all over the page
```suggestion
      <a href="#section-1" class="logo">
        <img
          src="./images/logo.png"
          alt="logo"
          class="logo__img"
        >
      </a>
```